### DISCUSSION TITLE
Same suggestion in different files doesn't show on files tab

### DISCUSSION THREAD

**Discussion by BaconGirl** [community/community#36013] [Pull Requests]
I suggested the same code change to 2 different files and I was planning to add those suggestions in a batch to commit them together.
On the 'Conversation' tab I can see both suggestions but the 'add suggestion to batch' is disabled.
![image](https://user-images.githubusercontent.com/6996532/195565643-3e9d8337-a31a-4a85-844f-50a97d9d8851.png)

When i go to 'Files' tab I see my suggestion in only one of the files making it impossible to add the second one to the batch and commit them together.

I also tried to access the suggestion via the conversation dropdown (see image below) but both reference to the first file somehow. Red underline for file name difference.
![image](https://user-images.githubusercontent.com/6996532/195572025-7e89bd08-5792-49a5-852f-96d8290077ed.png)

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @BaconGirl ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
